src/peft/tuners/lora/tp_layer.py

```diff
@@ -81,6 +81,14 @@ def __init__(
 
         self.is_target_conv_1d_layer = False
 
+    @property
+    def is_paralle_a(self):
+        # See https://github.com/huggingface/peft/pull/1439 for more details
```

Comment:
Let's add a `# TODO remove in version X.Y` to these types of deprecations to make it easier to find them in the future.

Reply:
Done ! I propose to remove it in 2 minor releases